examples/multibody/deformable_torus/deformable_torus.cc

```diff
@@ -149,17 +276,36 @@ int do_main() {
 
   auto [plant, scene_graph] = AddMultibodyPlant(plant_config, &builder);
 
+  const auto M = SpatialInertia<double>::SolidCubeWithMass(0.1, 0.05);
+  ModelInstanceIndex model_instance = plant.AddModelInstance("instance");
+  const auto& body = plant.AddRigidBody("body", model_instance, M);
+  Capsule box{0.02, 0.1};
+  IllustrationProperties cup_illustration_props;
+  cup_illustration_props.AddProperty("phong", "diffuse",
+                                     Vector4d(0.9, 0.1, 0.1, 0.8));
+  plant.RegisterVisualGeometry(body, RigidTransformd::Identity(), box,
+                               "cup_visual", cup_illustration_props);
+  const auto& prismatic_joint = plant.AddJoint<PrismaticJoint>(
+      "translate_z_joint", plant.world_body(), std::nullopt, body, std::nullopt,
+      Vector3d::UnitZ());
+  auto actuator_index =
+      plant.AddJointActuator("prismatic joint actuator", prismatic_joint)
+          .index();
+
+  plant.get_mutable_joint_actuator(actuator_index)
+      .set_controller_gains({1e6, 1});
+
   /* Minimum required proximity properties for rigid bodies to interact with
    deformable bodies.
    1. A valid Coulomb friction coefficient, and
-   2. A resolution hint. (Rigid bodies need to be tessellated so that collision
-   queries can be performed against deformable geometries.) */
+   2. A resolution hint. (Rigid bodies need to be tessellated so that
+   collision queries can be performed against deformable geometries.) */
   ProximityProperties rigid_proximity_props;
   /* Set the friction coefficient close to that of rubber against rubber. */
-  const CoulombFriction<double> surface_friction(1.15, 1.15);
+  const CoulombFriction<double> surface_friction(0.15, 0.15);
   AddContactMaterial({}, {}, surface_friction, &rigid_proximity_props);
   rigid_proximity_props.AddProperty(geometry::internal::kHydroGroup,
```

Comment:
Potentially a silly question, but does this example continue to work fine if the gripper is made to be hydroelastic? We're currently changing the collision geometry of our suction cups to use hydroelastic contact.